src/plugins/plugin.decimation.js

```diff
@@ -158,7 +158,7 @@ function cleanDecimatedDataset(dataset) {
     const data = dataset._data;
     delete dataset._decimated;
     delete dataset._data;
-    Object.defineProperty(dataset, 'data', {value: data});
+    dataset.data = data;
```

Comment:
This would actually skip the cleanup, because the configured `set` function would store the data in this plugin.
So I think the defineProperty needs just some additional parameters.

Reply:
Good catch! Completely missed that. Just adjusted accordingly.